src/restconfig/src/test/java/org/geoserver/catalog/rest/FeatureTypeTest.java

```diff
@@ -438,8 +440,8 @@ public void testDelete() throws Exception {
         }
         if( catalog.getResourcePool().getDataStoreCache().containsKey( dataStoreId ) ){
             DataAccess dataStore = catalog.getResourcePool().getDataStoreCache().get( dataStoreId );
-            List<String> names = dataStore.getNames();
-            assertTrue( names.contains("PrimativeGeoFeature"));
+            List<Name> names = dataStore.getNames();
+            assertTrue( names.contains(name));
```

Comment:
'Fixed' the spelling mistake here by making the test more robust (by using the actual feature type name)